### DISCUSSION TITLE
Atom.io Markdown Wrapper package - replacement?

### DISCUSSION THREAD

**Discussion by mickfuzz** [pulsar-edit/.github#177] [Community Package Help] [answered]
Hi there - Greetings to all community member for this great project! Kudos to all. 

I was using - https://github.com/stoe/atom-markdown-wrapper
for the following purposes. (see below)
It's now archived and doesn't work on Pulsar. 

Is there a different package or another way to replicate these shortcuts. 

Bold

    Select the text
    Click ctrl+b (cmd+b on Macs) and the selected text will be marked bold (**bold**).

Italic

    Select the text
    Click ctrl+i (cmd+i on Macs) and the selected text will be marked italic (_italic_).
    Click ctrl+b (cmd+b on Macs) and the selected text will be marked bold (**bold**).

Strikethrough

    Select the text
    Click ctrl+s (ctrl+i on Macs) and the selected text will be marked strikethrough (~~strikethrough~~).

**Comment by Daeraxa** [comment, marked as answer]
https://web.pulsar-edit.dev/packages/markdown-writer?

I've not tried it but it was updated more recently and seems to have had a lot of users.

**Comment by mickfuzz** [comment]
Yes that worked well. 
I had to do this to make it work I think. 
https://github.com/zhuochun/md-writer/issues/228

**Reply by mickfuzz** [reply]
many thanks!
